tests/auth_basic_test.py

```diff
@@ -64,41 +65,63 @@ def test_signup_route(client, capture_templates, setup_logger):
         raise
 
 
-def test_signup_post_invalid_email(client, capture_templates, setup_logger):
+param_list = [
+    ("bad_email", "TestP@ssword!", HTTPStatus.OK),  # bad email
+    ("test@example.com", "wrongpassword", HTTPStatus.OK),  # wrong password
+    # SQL injections
+    ("test@example.com' OR '1'='1' --", "TestP@ssword!", HTTPStatus.OK),
+    (
+        "test@example.com",
+        "wrongpassword' UNION SELECT 1, username, password FROM users --",
+        HTTPStatus.OK,
+    ),
+]
+
```

Comment:
Moving the `param_list` would be a better look, imo. It isn't getting used anywhere else and if you continued this pattern you'd have multiple `param_list`s.

Reply:
updated!